Yafc.Model/Analysis/Milestones.cs

```diff
@@ -33,20 +29,19 @@ public bool IsAccessibleAtNextMilestone(FactorioObject obj) {
         if (milestoneMask[0]) {
             return false;
         }
-        // TODO Always returns false -> milestoneMask is a power of 2 + 1 always has bit 0 set, as x pow 2 sets one (high) bit, so the + 1 adds bit 0, which is detected by (milestoneMask & 1) != 0
+        // TODO Always returns false -> milestoneMask is a power of 2 + 1 always has bit 0 set, as x pow 2 sets one (high) bit,
+        // so the + 1 adds bit 0, which is detected by (milestoneMask & 1) != 0
         // return ((milestoneMask - 1) & (milestoneMask - 2)) == 0; // milestoneMask is a power of 2 + 1
         return false;
     }
 
-    public Bits GetMilestoneResult(FactorioId obj) {
+    public Bits GetMilestoneResult(FactorioId obj) =>
         // Return a copy of Bits
-        return new Bits(milestoneResult[obj]);
-    }
+        new Bits(milestoneResult[obj]);
```

Comment:
Possibly make this, and the one below, a one-liner?

Reply:
If I remember correctly, this one was left as a multi-liner because of the comment above the changed line. It can be adjusted, no problem.

Reply:
Ah yes. But the comment can go above or inline with the one-liner either way.

Reply:
Addressed in https://github.com/shpaass/yafc-ce/pull/270/commits/4b363c889b5fc5c22178b6a3affbb66e08f80b8f